app/frontend/src/search/ui/sort.js

```diff
@@ -1,38 +1,25 @@
 export const renderSortSelect = (renderOptions, isFirstRender) => {
```

Comment:
Are there any better options in terms of renaming this function (while we're here)? 'Render sort select' doesn't mean much to me as an English phrase because I don't know what a sort select is.